google/cloud/forseti/services/model/service.py

```diff
@@ -133,16 +137,30 @@ def GetModel(self, request, _):
         """
 
         model = self.modeller.get_model(request.identifier)
-
+        created_at_str = self._get_model_created_at_str(model)
         if model:
             return model_pb2.ModelDetails(name=model.name,
                                           handle=model.handle,
                                           status=model.state,
+                                          createdAt=created_at_str,
                                           description=model.description,
                                           message=model.message,
                                           warnings=model.warnings)
         return model_pb2.ModelDetails()
 
+    @staticmethod
+    def _get_model_created_at_str(model):
+        """Get model created_at datetime in string format.
+        Example return value: '2011-11-03 18:21:26'
+
+        Args:
+            model (Model): Model dao object.
+
+        Return:
+            str: created_at datetime in string format.
+        """
+        return model.created_at_datetime.now().strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
Can we use one of the centralized timestamp string from here?  `DEFAULT_FORSETI_HUMAN_TIMESTAMP`

https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-dev/google/cloud/forseti/common/util/string_formats.py#L49

Reply:
Updated.